### DISCUSSION TITLE
Export and import of Timers

### DISCUSSION THREAD

**Discussion by ronniedroid** [FossifyOrg/General-Discussion#637] [Ideas]
Hey,

I want to submit a pull request to add the ability to export and import timers as I do have many of them and it is a bit annoying to set them up all again when I wipe my phone or switch ROMs. 

In case the idea is accepted, I have already submitted a PR for exporting and importing Alarms, if you like the idea, should I merge them both and make it one operation to export and import the alarms and the timers?

**Comment by ronniedroid** [comment]
@naveensingh what do you think?

**Comment by naveensingh** [comment]
I think there should be a single option to export both alarms and timers.
